uPortal-soffit-connector/src/main/java/org/apereo/portal/soffit/connector/SoffitConnectorController.java

```diff
@@ -57,6 +57,10 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.portlet.bind.annotation.RenderMapping;
 
+/**
+ * @since 5.0
+ * @author drewwills
```

Comment:
I'd favor a style of not introducing `@author` annotations. They don't age well in shared-owner, shared-maintenance, shared-development code. The Git history is sufficient indication of the author.